src/svxlink/svxlink/locale.tcl

```diff
@@ -129,37 +130,58 @@ proc playNumber {number} {
 # Say the time specified by function arguments "hour" and "minute".
 #
 proc playTime {hour minute} {
+  global time_format
```

Comment:
You can implement this without modifying the C++ code since any configuration variable is available in the TCL code as CFG_varname. So if you specify TIME_FORMAT in the config you can access it as CFG_TIME_FORMAT in the TCL code. E.g.:
```
variable Logic::CFG_TIME_FORMAT
puts "### TIME_FORMAT=$CFG_TIME_FORMAT"
```
This also make this change tied to the English locale.tcl rather than "forcing" it on all langpacks.

Reply:
I fixed up the documentation issues, which also resolved the conflict.
~~~~~~~~
I do not understand how this change is "forcing" anything on all language packs,  the default behavior is maintained (12 hour is the default value). If the config variable is not the acceptable option of "24", it plays the 12 hour behavior, same as before.

If the other language packs don't bring in the new variable & code into their locale.tcl, wouldn't this new change be ignored anyways?

Reply:
Yes, but there is no reason to modify the C++ code in any case. What I meant with forcing (maybe the wrong word) was just that some languages may not have the concept of 12/24 hour format so it's language pack specific. Then it may be confusing if people think that they can use the TIME_FORMAT config variable for any langpack.

A little caveat is that the option still have to be documented somewhere and since there is no concept for documenting a langpack at the moment the only place is the svxlink.conf manual page. I guess a note about "may not work for all langpacks" will have to do for now.